### PR TITLE
Title and subtitle atoms adjustments

### PR DESCRIPTION
**Pull request by mveres**
https://backlight.dev/edit/1SuMaEt5uxgp62VC8WVP/hero/src/index.vue?branch=atomic-design-web@tLlGPKdxk2VRYywCKUXNaV8PFt62

### CODE REVIEW

**Review by gqio — CHANGES_REQUESTED** [review]
I am very sorry but we can’t use title as className. It will leak in all class title we already have in the app. Please use another className

File: title/src/title.css [hunk 1]
```diff
@@ -2,12 +2,10 @@
 @tailwind utilities;
 
 @layer components {
-  .hero .title {
+  .title {
```
Comment (gqio) [inline title/src/title.css:5]: it’s not a title… It’a BigTitle if you want. but definitely not a title. I think it’s too generic title

File: subtitle/src/subtitle.css [hunk 2]
```diff
@@ -2,18 +2,13 @@
 @tailwind utilities;
 
 @layer components {
-  .hero .subtitle {
-    @apply mt-3
-		text-base
```
Comment (gqio) [inline subtitle/src/subtitle.css:7]: Same I think it’s too generic subtitle. Please use something indicating it’s a big sub title

File: title/src/title.css (hunk 1 above)
Comment (mveres) [reply title/src/title.css:5]: Ok, will do.
I checked where else `title` is used on the landing page and I thought we were safe... but indeed it's too generic.

**Review by gqio — APPROVED** [review]

**Pull request merged**